mkdocs.yml

```diff
@@ -33,6 +33,8 @@ extra:
     alias: true
 
 markdown_extensions:
+  - admonition
+  - codehilite
```

Comment:
What are these extensions for? Can we add inline comments for their purpose please if possible?

Reply:
Example
- https://astronomer.github.io/dag-factory/dev/contributing/howto/#2-activate-the-local-python-environment
vs
- https://squidfunk.github.io/mkdocs-material/reference/admonitions/